### PR TITLE
Add hostile militia retorts

### PR DESCRIPTION
**Pull request by Ferociousfeind**
Add another "hostile militia" phrase, because the prisons in the Rim have some rather empty cells.

### CODE REVIEW

File: data/hails.txt [hunk 1]
```diff
@@ -3112,6 +3112,62 @@ phrase "hostile militia"
...
+		"Flood"
+		"Longjump"
+		"Glaze"
+		"Deep"
+		"Clink"
+		"Winter"
+		"Solace"
+		"Twinstar"
+		"Sundrinker"
+		"Trinken"
```
Comment (Bladewood) [inline data/hails.txt]: hmm...

Comment (jafdy) [reply data/hails.txt]: Had you been drinken when you wrote that?

Comment (MCOfficer) [reply data/hails.txt]: skål

**Comment by endless-sky** [comment]
"The prisons are rather empty" seems like a rather vague way of phrasing the threat - maybe something like "I hope you like prison food" would be clearer.

(Also, the fact that there isn't an actual game mechanic where you can be boarded and take prisoner could make this message confusing to players.)

**Comment by Pointedstick** [comment]
I agree with @endless-sky. @Ferociousfeind can you re-work this PR according to review comments, or else close it?

File: data/hails.txt [hunk 2]
```diff
@@ -3112,6 +3112,64 @@ phrase "hostile militia"
...
+		"We "
+	word
+		"hope "
+		"wonder if "
+	word
+		"you "
+		"criminals like you "
+		
+	word
+		"like"
```
Comment (Amazinite) [inline data/hails.txt]: -> `"like "`

Comment (Ferociousfeind) [reply data/hails.txt]: Oh, I see your point now. How didn't I catch that??

Comment (jafdy) [reply data/hails.txt]: Same way everyone else doesn't catch mistakes, too many lines for mistakes to slip into. This is why we review everything.

**Pull request merged**